gridfs/grid_file.py

```diff
@@ -357,12 +357,14 @@ def write(self, data: Any) -> None:
         except AttributeError:
             # string
             if not isinstance(data, (str, bytes)):
-                raise TypeError("can only write strings or file-like objects")
+                raise TypeError("can only write strings or file-like objects") from None
```

Comment:
What are the pros of adding `from None`? Are there any cons?

Reply:
It affects whether the previous error shows up in the traceback or not.